server/mailet/mailets/src/test/java/org/apache/james/transport/matchers/IsInDropListTest.java

```diff
@@ -0,0 +1,134 @@
+package org.apache.james.transport.matchers;
```

Comment:
You are missing the Apache V2 license

Reply:
In my opinion, such things should be checked automatically using the maven-checkstyle-plugin.
https://github.com/apache/james-project/pull/2218